pom.xml

```diff
@@ -22,6 +22,12 @@
             <email>stuart.gunter@composable.co.uk</email>
             <url>http://www.stuartgunter.org</url>
         </developer>
+        <developer>
```

Comment:
This should be under `<contributors>`. The `<developers>` section is for core committers. Please see https://maven.apache.org/pom.html#Contributors for more info.